src/rezplugins/shell/pwsh.py

```diff
@@ -1,25 +1,12 @@
 """Windows PowerShell 6+"""
 
-from rez.config import config
-from rez.rex import RexExecutor, OutputStyle, EscapedString
 from rez.shells import Shell
-from rez.utils.system import popen
 from rez.utils.platform_ import platform_
-from rez.backport.shutilwhich import which
-from functools import partial
-from subprocess import PIPE
-import os
-import re
+from rezplugins.shell.powershell import PowerShellBase
+from rezplugins.shell.sh import SH
```

Comment:
..but this should be changed

Reply:
Hope that comment makes it clearer why we use SH here.